README.md

```diff
@@ -294,7 +294,7 @@ on:
       - main
   
 jobs:
-  call-release-workflow:
+  call-release-checklist-workflow:
     uses: ASFHyP3/actions/.github/workflows/reusable-release-checklist-comment.yml@v0.7.1
     permissions:
       pull-requests: write
```

Comment:
`permissions` turned out to be unnecessary for repositories with GITHUB_TOKEN permissions set to permissive, and prevents the action from working for private repositories. I'd suggest removing it from the example.

Reply:
@asjohnston-asf happy to review a PR :upside_down_face: